### DISCUSSION TITLE
No Way to Access Community Store/Page as Minecraft Character

### DISCUSSION THREAD

**Discussion by RoyalT20** [gameserverapp/Platform#934] [Suggestions]
**Describe the bug**
There is currently no way to access the community page/store as a character connected to a Minecraft account. Even after connecting a Microsoft account the Minecraft character is seen in the GSA character list as a distinct account. I have not determined any way as an admin or player to connect these accounts. This prevents purchases by and automated deliveries to Minecraft players.

**To Reproduce**
1. Sign into the community store/page with a Microsoft account.
2. Log into a GSA hosted Minecraft world while signed into the same Microsoft Account.
3. The accounts are not linked in any way on the community store/page and show separately in admin dashboards.
4. There is no way to login or to make a purchase as the Minecraft character.

**Expected behavior**
When accessing a GSA hosted world while signed in to a Microsoft account, a player should then be able to access the community store/page while signed into that Microsoft account and make purchases that can be delivered to that character.

**Comment by RoyalT20** [comment]
I'm actually sad to see this moved this morning as it being opened as an issue rather than a suggestion was deliberate. As currently implemented, the GSA platform does NOT support monetization or even community management in Minecraft as claimed in several places on the GSA website. I could as easily use the console to manage my Minecraft servers as GSA in its current state. Unhappy language aside, I would like to discuss funding the development needed to fix this issue. As far as I can tell the solution is as simple as an API call using a Microsoft platform that I have previously provided the link to, but as GSA is a proprietary platform this is not a change I can make for you. I look forward to your response.

**Reply by mvaessen** [reply]
Hi, the issue section is for broken features & bugs. Suggestions are for missing features. In this case the feature is missing, not broken. Minecraft login has never been supported, but I'm hoping to add that at some point, now that Microsoft added api's for us to work with.

**Reply by RoyalT20** [reply]
Fair enough, thank you for the quick response and explanation! Is there any way I can help with the development of this feature?

**Reply by mightyhuff2** [reply]
do it do it do it do it

**Reply by KrashKourse** [reply]
I mean Microsoft does know how to send it to a character with the API so it should be doable. It would be nice to see this in the future.

**Comment by Decollates** [comment]
Given that my own community is starting to branch off into Minecraft, I am definitely invested in this development.

**Comment by mightyhuff2** [comment]
I am extremely interested in seeing this happen. This would be great.

**Comment by RoyalT20** [comment]
Sir Max, should you see this, have an answer, and also feel like sharing it, do you have an approximate roadmap of when we might see this feature rolled out? Not to tie you to an answer or anything, I simply have some hardware reserved for Minecraft that I am trying to decide what to do with. If you think we will see this within a month I will likely keep it where it is at, if you think it will be longer then that I would like to repurpose that hardware for the time being. Thank you!

**Comment by mvaessen** [comment]
duplicate of #640

**Comment by mvaessen** [comment]
Minecraft login is now available
